cms/static/js/spec/views/login_studio_spec.js

```diff
@@ -0,0 +1,30 @@
+define(['jquery', 'js/factories/login', 'common/js/spec_helpers/ajax_helpers', 'common/js/components/utils/view_utils'],
+function($, LoginFactory, AjaxHelpers, ViewUtils) {
+    'use strict';
+    describe("Studio Login Page", function() {
+        var submit_button;
+
+        beforeEach(function() {
+            loadFixtures('login.underscore');
+            LoginFactory("/home/");
+            submit_button = $('#submit');
+        });
+
+        it('It will disable the submit button once its clicked', function() {
+            spyOn(ViewUtils, 'redirect').andCallFake(function(){});
+            var requests = AjaxHelpers.requests(this);
+            expect(submit_button).not.toHaveClass('is-disabled');
+            submit_button.click();
+            AjaxHelpers.respondWithJson(requests, {'success': true});
+            expect(submit_button).toHaveClass('is-disabled');
```

Comment:
@adampalay there are other instances in repo where they using the same method ```toHaveClass``` mentioned at https://github.com/velesin/jasmine-jquery for jquery,So still i need the above to replace with ```toHaveBeenCalledWith``` ?